app/Http/Kernel.php

```diff
@@ -79,24 +79,6 @@ class Kernel extends HttpKernel
         'verified' => \Illuminate\Auth\Middleware\EnsureEmailIsVerified::class,
     ];
 
-    /**
-     * The priority-sorted list of middleware.
-     *
-     * This forces non-global middleware to always be in the given order.
-     *
-     * @var array
-     */
-    protected $middlewarePriority = [
-        \Illuminate\Session\Middleware\StartSession::class,
-        \Illuminate\View\Middleware\ShareErrorsFromSession::class,
-        \App\Http\Middleware\LegacyExternalAuth::class,
```

Comment:
I think this needs to be left because LegacyExternalAuth needs to be before Authenticate

Reply:
Laravel removed it upstream, but seems its still supported behind the scences